src/ex6/client/src/components/footer/footer.jsx

```diff
@@ -0,0 +1,35 @@
+
+import React from "react";
+import PropTypes from "prop-types";
+import "./footer.css";
+import ItemClient from "../../services/item_client.js";
+import { Button} from "monday-ui-react-core";
+
+const Footer = ({setTodoList,listTodo}) => {
+  const itemClient = new ItemClient();
+
+  const deleteListToDo = async () => {
+      await itemClient.deleteAllTodo();
+      setTodoList([]);
+  };
+
+  const showButtonClearAll = () => {
+    if (listTodo.length > 0){
+      return <Button type="button" onClick={()=> deleteListToDo()}>Clear All</Button> 

```

Comment:
you can just state onClick={deleteListToDo} instead, it will be cleaner and better performant